src/Avalonia.Visuals/Media/DrawingGroup.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Avalonia.Collections;
+using Avalonia.Metadata;
+
+namespace Avalonia.Media
+{
+    public class DrawingGroup : Drawing
+    {
+        [Content]
+        public AvaloniaList<Drawing> Children { get; } = new AvaloniaList<Drawing>();
+
+        public static readonly StyledProperty<double> OpacityProperty =
```

Comment:
Yeah, we used to have them, but there were complaints that it was too strict and caused too much distraction... Personally I'd like to add them back but that would hit resistance. Not sure what the best way to handle style-related things is, but for now I just try to point them out in PRs, which isn't ideal as they're not really documented or enforced.